### PR TITLE
fixed broken set_pedigree_capture_kit_info.t, #429

### PR DESCRIPTION
**Pull request by jemten**

### CODE REVIEW

**Comment by jemten** [comment]
Hmm, might have misunderstood the function a bit. Does it need to return undef for this case?

File: t/set_pedigree_capture_kit_info.t [hunk 1]
```diff
@@ -234,8 +234,12 @@ set_pedigree_capture_kit_info(
         user_supply_switch_href => \%user_supply_switch,
     }
 );
-is( $active_parameter{exome_target_bed},
-    undef, q(No capture kit from cmd, config or pedigree) );
+
+is(
```
Comment (henrikstranneheim) [inline t/set_pedigree_capture_kit_info.t]: It should return undef to signify that the user has used a capture kit where there is no akronym shortcut that can be used. Actually setting to latest is done elsewhere and occurs when the user did not set any capture kit info at all in this run or previous runs. It is a bit confusing I know.

File: lib/MIP/Set/Pedigree.pm [hunk 2]
```diff
@@ -195,11 +195,11 @@ sub set_pedigree_capture_kit_info {
         }
 
         ## No recorded capture kit from pedigree or previous run
-	if ( not $capture_kit ) {
+	    if ( not $capture_kit ) {
```
Comment (henrikstranneheim) [inline lib/MIP/Set/Pedigree.pm]: Is the indent really correct?

Comment (jemten) [reply lib/MIP/Set/Pedigree.pm]: It isn't. Though it looks coorect on my local copy...

File: t/set_pedigree_capture_kit_info.t (hunk 1 above)
Comment (jemten) [reply t/set_pedigree_capture_kit_info.t]: OK. Right now it looks like it is set to latest in the set_pedigree_capture_kit_info subroutine

Comment (henrikstranneheim) [reply t/set_pedigree_capture_kit_info.t]: Yes, you are rigth I confused it with get_capture_kit. When there is no info from cmd, config or pedigree we should default to "last" to set the default.

File: lib/MIP/Set/Pedigree.pm (hunk 2 above)
Comment (henrikstranneheim) [reply lib/MIP/Set/Pedigree.pm]: ok

**Comment by henrikstranneheim** [comment]
👍

**Pull request merged**